README.md

```diff
@@ -2,10 +2,29 @@
 
 ![bragi-icon](public/bragi-icon.png)
 
+## Description
+
+On parties' people want to hear their favorite songs. The idea behind **Bragi 3000** is to give each person a chance to
+hear their favorite song. This is achieved by letting people battle with each other for their song to be added to the
+party playlist/queue. These battles should be simple and fun and the winner's song gets added. One example for such a
+game is tic tac toe and could be changed to a different game. The song of the loser will be discarded and be temporarily
+be banned from being added to the playlist/queue.
+
+## Finished Features
+- Firebase and Spotify authentication
+- Song controller ( shows current playing song on spotify and can pause/resume it )
+- Song selector ( allows users to search and select a song they want to battle for )
+- MiniGame ( tic tac toe as a way to battle for which song gets added )
+
+## Planned Features
+- Landing Page
+- Adding song to playlist/queue and banning other song
+- Styling
+
```

Comment:
I think these can be a bit more elaborate, mentioning both state and components. Like "adding song to playlist", can be split into multiple things:

- Adding winning songs to the active playlist/queue
- Save winning and losing song in the state
- Adding components that show the won and lost songs from the state
- Wiring all components so that they correctly respond to things that happened within each other (song selector should reset when a game is won, for example)

Reply:
And also take a look at the issues here on github. Things like improving error handling could also be on the list